src/redux/initialState.js

```diff
@@ -126,6 +126,7 @@ export default {
       AnnotationCreateDistanceMeasurement: { dataElement: 'distanceMeasurementToolButton', title: 'annotation.distanceMeasurement', img: 'ic_annotation_distance_black_24px', group: 'measurementTools', showColor: 'active' },
       AnnotationCreatePerimeterMeasurement: { dataElement: 'perimeterMeasurementToolButton', title: 'annotation.perimeterMeasurement', img: 'ic_annotation_perimeter_black_24px', group: 'measurementTools', showColor: 'active' },
       AnnotationCreateAreaMeasurement: { dataElement: 'areaMeasurementToolButton', title: 'annotation.areaMeasurement', img: 'ic_annotation_area_black_24px', group: 'measurementTools', showColor: 'active' },
+      AnnotationCreateEllipseMeasurement: { dataElement: 'ellipseMeasurementToolButton', title: 'annotation.ellipseMeasurement', img: 'ic_annotation_circle_black_24px', group: 'measurementTools', showColor: 'active' },
```

Comment:
Do you think we should use another icon for the new tool? Maybe the icon should have dashed border(like measure polyline), or it's a circle that's filled with lines(like measure polygon).